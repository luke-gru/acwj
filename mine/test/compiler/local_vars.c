int a; int b; int c;

int main()
{
  char z; int y; int x;

  x= 10;  y= 20; z= 'c';
  a= 5;   b= 15; c= 25;

  printint(x);
  printint(y);
  printchar(z);
  printchar('\n');

  printint(a);
  printint(b);
  printint(c);
  return(0);
}
