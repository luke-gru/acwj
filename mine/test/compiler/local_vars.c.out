10
20
c
5
15
25
